src/config.rs

```diff
@@ -332,6 +335,8 @@ pub(crate) struct GooseDefaults {
     pub websocket_host: Option<String>,
     /// An optional default for port WebSocket Controller listens on.
     pub websocket_port: Option<u16>,
+    /// Allow for internal testing when no certificates are available but still using https
```

Comment:
Please use the same format as the above:
`An optional default for not validating https certificates.`